horace_core/sqw/file_io/@combine_sqw_pix_job/combine_sqw_pix_job.m

```diff
@@ -10,6 +10,10 @@
         is_finished_  = false;
         finalizer_ =[];
         open_files_id_ = [];
+        % The number of auxiliary worker, not participating in reading data
+        % It can be 1 single writer-combiner/mutliple readers
+        % or 2 (1 writer, 1 combiner/multiple readers)
+        reader_id_shift_ = 1;
```

Comment:
Is it possible to do named constants or enumerations for the values 1 and 2 here so the code reader can see immediately what has been chosen?

Reply:
It is possible, though not very convenient. You roughly define a class, with C-style enumeration (see MESS_CODES.m) or more complex class connecting strings and numbers (config classes). Its not done there, as I suspect there would be no need in keeping both options. I expect that after proper profiling only one job configuration will be proven more performant -- option 2. I may be wrong and if any each configuration will be best for some situation, we will do this and add the appropriate text option to `hpc_config` class. At the moment it is only internal option to change at debugging